src/format.ts

```diff
@@ -41,8 +42,8 @@ export async function format(
   } else {
     const result = await checkFormat(srcFiles);
     if (!result) {
-      options.logger.log(
-          'clang-format reported errors... run `gts fix` to address.');
+      options.logger.log(`clang-format ${
+          clangFormatVersion} reported errors... run \`gts fix\` to address.`);
```

Comment:
When is this version information useful?

Reply:
`clang-format` doesn't follow semver. If we fix the version number this isn't an issue, but it is still good to know. Even when the issue isn't clang-format, it is nice to be able to tell that at a glance.